verification/src/compiler/version.rs

```diff
@@ -0,0 +1,115 @@
+use semver::Version;
+use std::{fmt::Display, str::FromStr};
+
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub struct ReleaseVersion {
+    pub version: Version,
+    pub commit: String,
+}
+
+impl ReleaseVersion {
+    pub fn new(version: Version, commit: String) -> Self {
+        Self { version, commit }
+    }
+}
+
+/// Parses release version from string formated as
+/// `solc-v*VERSION*+commit.*COMMITHASH*`, example
+/// `solc-v0.8.9+commit.e5eed63a`
```

Comment:
The comments are not shown as a doc. Most probably, you would like to write them over the `from_str` function